### PR TITLE
Fixed #1200 -- Add more languages search support

### LINKED ISSUE #1200: Add more languages search support
Original multi-language search support was based on PostgreSQL 10, but after the P[ostgreSQL 12 update](https://github.com/django/djangoproject.com/pull/1151) we can add new supported language in the search configuration.

### PR DESCRIPTION
**Pull request by pauloxnet**
Part of #1031

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ services:
     depends_on:
       - db
   db:
-    image: postgres:11.2-alpine
+    image: postgres:12-alpine
```
Comment (CuriousLearner) [inline docker-compose.yml:26]: Ah, good catch!

Seems like the container for Github Actions [was updated here](https://github.com/django/djangoproject.com/pull/1151), but not the docker-compose file itself.

**Review by CuriousLearner — COMMENTED** [review]
Overall, looks good to me. Though I'm wondering if we need to test these config languages?

It seems to be [used here](https://github.com/django/djangoproject.com/blob/d0dba2554accefebcc4fed3f9c7645d8405a79e8/docs/models.py#L174), though I'm not sure if an explicit test is needed.

**Comment by pauloxnet** [comment]
There is already a specific test for multilingual search using this config languages.
https://github.com/django/djangoproject.com/blob/main/docs/tests.py#L490
But I don't think it would be an added value to have a specific test for each new language we add here because it is supported by PostgreSQL.

**Review by CuriousLearner — APPROVED** [review]
Sounds good. Let's get this in! :)

**Pull request merged**